habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/BuildDeploymentArtifactsMojo.java

```diff
@@ -112,6 +127,10 @@ public void doExecute() throws MojoExecutionException, MojoFailureException {
 
             setUpPlaceholderFileAsMavenArtifact();
         }
+
+        if(cacheBuildWheelFiles){
+            cacheWheelFiles();
```

Comment:
S: Traditionally this kind of action would happen during the `install` phase instead of the `package` phase. If we feel strongly it should stay in the package phase by default, we should call out it out in the docs so it doesn't trip people up. Otherwise, I'd just introduce a new Mojo whose sole responsibility is to install the wheel files into the poetry cache.

Reply:
Possible to push this to a follow-on as well. "As a Habushu user, I want to be able my wheels installed to the local cache separately from the package step, so I have more control over the build lifecycle."